dev-client/.depcheckrc

```diff
@@ -14,6 +14,7 @@
 		"react-native-gradle-plugin",
 		"react-redux",
 		"terraso-mobile-client",
-		"uuid"
+		"uuid",
+                "@testing"
```

Comment:
question: should this me @testing/xxx?

Reply:
Yes, good catch. If I put `@testing/*` it works.